### PR TITLE
Add clean workspace.

### PR DESCRIPTION
**Pull request by ghost**
I propose to remove the zip file at the end of the installation.

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -45,3 +50,8 @@
 
 - name: Make sure Sonar is responding on the configured port.
   wait_for: port=9000 delay=3 timeout=300
+
+- name: Clean workspace.
+  file:
+    path: "{{ workspace }}/{{ sonar_version_directory }}.zip"
+    state: absent
```
Comment (geerlingguy) [inline tasks/main.yml]: Please keep the newline at EOF

Comment (ghost) [inline tasks/main.yml]: Sorry for the inconvenience. Correction done.

**Comment by ghost** [comment]
Rebased.

**Comment by goloxy** [comment]
Rebased.

**Comment by ghost** [comment]
Rebased.

**Pull request closed without merging**